actors/user/src/test/java/org/sunbird/user/UserRoleActorTest.java

```diff
@@ -0,0 +1,180 @@
+  private static CassandraOperationImpl cassandraOperation;
+  private static RoleDaoImpl roleDao;
+  private static ActorSelection actorSelection;
+  private static CompletionStage completionStage;
+  private static ActorRef actorRef;
+  private static InterServiceCommunication interServiceCommunication;
+  private static Organisation organisation;
+  private static Response response;
+  private static Map<String, Object> map;
+
+  @Before
+  public void beforeEachTest() {
+
+    PowerMockito.mockStatic(ServiceFactory.class);
+    cassandraOperation = mock(CassandraOperationImpl.class);
+    when(ServiceFactory.getInstance()).thenReturn(cassandraOperation);
+    PowerMockito.mockStatic(RoleDaoImpl.class);
+    roleDao = Mockito.mock(RoleDaoImpl.class);
+    when(RoleDaoImpl.getInstance()).thenReturn(roleDao);
+    system = ActorSystem.create("system");
+    PowerMockito.mockStatic(ServiceFactory.class);
+    cassandraOperation = mock(CassandraOperationImpl.class);
+    when(ServiceFactory.getInstance()).thenReturn(cassandraOperation);
+    actorSelection = Mockito.mock(ActorSelection.class);
+    PowerMockito.mockStatic(BaseMWService.class);
+    completionStage = Mockito.mock(CompletionStage.class);
+    PowerMockito.mockStatic(RequestRouter.class);
+    actorRef = Mockito.mock(ActorRef.class);
+
+    PowerMockito.mockStatic(InterServiceCommunicationFactory.class);
+    interServiceCommunication = Mockito.mock(InterServiceCommunication.class);
+    when(InterServiceCommunicationFactory.getInstance()).thenReturn(interServiceCommunication);
+    organisation = Mockito.mock(Organisation.class);
+    response = Mockito.mock(Response.class);
+    map = Mockito.mock(Map.class);
+  }
+
+  @Test
+  public void testGetUserRoleSuccess() {
+
+    TestKit probe = new TestKit(system);
+    ActorRef subject = system.actorOf(props);
+    Response response = getCassandraResponse();
+    when(cassandraOperation.getAllRecords(Mockito.anyString(), Mockito.anyString()))
+        .thenReturn(response);
+
+    Request reqObj = new Request();
+    reqObj.setOperation(ActorOperations.GET_ROLES.getValue());
+    subject.tell(reqObj, probe.getRef());
+
+    Response res = probe.expectMsgClass(duration("10 second"), Response.class);
+    Assert.assertTrue(null != res && res.getResponseCode() == ResponseCode.OK);
+  }
+
+  @Test
+  public void testAssignRolesSuccessWithValidRole() {
+
+    TestKit probe = new TestKit(system);
+    ActorRef subject = system.actorOf(props);
+
+    when(BaseMWService.getRemoteRouter(Mockito.anyString())).thenReturn(actorSelection);
+
+    when(actorSelection.resolveOneCS(Duration.create(Mockito.anyLong(), "seconds")))
+        .thenReturn(completionStage);
+    when(RequestRouter.getActor(Mockito.anyString())).thenReturn(actorRef);
+
+    when(interServiceCommunication.getResponse(Mockito.anyObject(), Mockito.anyObject()))
+        .thenReturn(response);
+    when(response.get(Mockito.anyString())).thenReturn(map);
```

Comment:
how is this connected, we are returning a mocked map, there are no comparison, no operation over this map in further code?